adele-booking/src/main/java/com/microservicesteam/adele/booking/domain/BookingService.java

```diff
@@ -79,6 +85,7 @@ public void handleEvent(TicketsBooked ticketsBooked) {
                         .position(position)
                         .bookingId(ticketsBooked.bookingId())
                         .build()));
+        LOGGER.debug("Event published on websocket {}", ticketsBooked);
```

Comment:
At this point event is not published yet and `WebSocketEventPublisher` does the job for you. What you need here is on debug level: `Ticket repository updated`